lldb/source/Target/SwiftLanguageRuntimeImpl.h

```diff
@@ -116,8 +116,8 @@ class SwiftLanguageRuntimeImpl {
 protected:
   // Classes that inherit from SwiftLanguageRuntime can see and modify these
   Value::ValueType GetValueType(Value::ValueType static_value_type,
-                                const CompilerType &static_type,
-                                const CompilerType &dynamic_type,
+                                CompilerType static_type,
+                                CompilerType dynamic_type,
```

Comment:
Why did you remove the const reference?

Reply:
That's a cleanup that I sneaked in. On llvm.org I globally applied this change and didn't yet in the Swift-specific parts.